scripts/solve_network.py

```diff
@@ -2052,7 +2125,7 @@ def add_ci(n: pypsa.Network, year: str, config: dict, costs: pd.DataFrame) -> No
 
         snakemake = mock_snakemake(
             "solve_sector_network_myopic",
-            run="baseline-3H",
+            run="baseline-3H-emi-match",
```

Comment:
```suggestion
            run="baseline-3H",
```
you can revert this after your test